skietbaan_fe/src/components/RegisterMember.js

```diff
@@ -36,21 +42,54 @@ class App extends Component {
     });
   }
 
-  RegisterMember() {
+  SearchMember() {
    if(this.state.validForm)
    {
-    var RequestObject = {
-        "Username":this.state.usernameValue,
-    }
-      const BASE_URL = 'http://skietbaan.retrotest.co.za/';
-      fetch(`${BASE_URL}`,"/api/values", {
+      fetch("http://api.skietbaan.retrotest.co.za/api/Features/Search?Username="+this.state.usernameValue, {
       method: 'Get',	
         headers: {'Accept': 'application/json','Content-Type': 'application/json'},
-        body: JSON.stringify(RequestObject)}).then(function(response) {
-          return response.json();}).then(function(data) {}).catch(function() {});
+      }).then(function(response) {
+          return response.json();})
+          .then(function(data) {
+            if(data.memberID == null){
+              document.getElementById("membershipID").value = data.memberID;
+            }
+            else{
+              document.getElementById("membershipID").value = data.memberID;
+            }
+            let entDate = data.entryDate.split('T');
+            let keepDate = data.entryDate.split('T');
+            document.getElementById("entrydate").value = entDate[0];
+            let dateNew = keepDate[0].split('-');
+            let exDateYear = parseInt(dateNew[0],10) + 1;
+            document.getElementById("expdate").value = exDateYear + '-' + dateNew[1] + '-' + dateNew[2];
+          })
+          .catch(function() {});
     }	    
   }
 
+  UpdateMember() {
+    let RequestObject = {
+      "username": document.getElementById("usernameValue").value,
+      "memberID": document.getElementById("membershipID").value,
+      "entryDate": document.getElementById("entrydate").value + "T00:00:00",
+      "memberExpiry": document.getElementById("expdate").value + "T00:00:00"
+    }
+       fetch("http://api.skietbaan.retrotest.co.za/api/Features/Update" , {
+       method: 'Put',	
+         headers: {'Accept': 'application/json','Content-Type': 'application/json'},
+         body: JSON.stringify(RequestObject)
+       }).then(function(response) {
+          ToastStore.success(document.getElementById("usernameValue").value + " is successfully registered as a member ");
+          
```

Comment:
Clean up the empty lines and fix indentations

Reply:
i have done that